tests/test_utils.py

```diff
@@ -1,65 +1,70 @@
 from datetime import datetime
 from pathlib import Path
-from unittest.mock import call, patch
+from unittest.mock import patch
 
 import pytest
-from botocore.stub import ANY, Stubber  # TODO: fully transition to moto for S3 mocking
 from moto import mock_aws
 from moto.core import patch_client
 
 import opera_disp_tms.utils as ut
 
 
 @pytest.fixture
-def s3_stubber():
-    with Stubber(ut.S3_CLIENT) as stubber:
-        yield stubber
-        stubber.assert_no_pending_responses()
+def s3_bucket():
+    with mock_aws():
+        patch_client(ut.S3_CLIENT)
+
+        bucket_name = 'myBucket'
+        location = {'LocationConstraint': 'us-west-2'}
+
+        ut.S3_CLIENT.create_bucket(Bucket=bucket_name, CreateBucketConfiguration=location)
+
+        yield bucket_name
 
 
 def test_within_in_day():
```

Comment:
I assume that's a typo
```suggestion
def test_within_one_day():
```